chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/certifiedProduct/AmbulatoryModular2014Validator.java

```diff
@@ -91,8 +91,9 @@ protected void validateDemographics(PendingCertifiedProductDTO product) {
 							product.getErrorMessages().add("There was no test tool found matching '" + testTool.getName() + "' for certification " + cert.getNumber() + ".");
 						} else {
 							TestToolDTO tt = ttDao.getById(testTool.getTestToolId());
-							if(tt != null && tt.isRetired()) {
-								product.getErrorMessages().add("Cannot use test tool '" + tt.getName() + "' since it is retired. Please choose a different test tool for " + cert.getNumber() + ".");
+							if(tt != null && tt.isRetired() && product.getIcs() == false) {
+								product.getErrorMessages().add("Test Tool '" + testTool.getName() + "' for Certification Result '" + cert.getNumber() 
+								+ "' has been retired for product with ICS=false.");

```

Comment:
Can we change this line to something like `can not be used, as it is a retired tool, and this Certified Product does not carry ICS`? Anywhere we have `ICS=false` that phrase is too programmery for our end users